<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RiabKeywordsDialogBase</class>
 <widget class="QDialog" name="RiabKeywordsDialogBase">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>520</width>
    <height>475</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Risk in a Box - Keyword Editor</string>
  </property>
  <layout class="QGridLayout" name="gridLayout_2">
   <item row="1" column="0">
    <widget class="QLabel" name="lblCategory">
     <property name="text">
      <string>Category</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1" colspan="2">
    <layout class="QHBoxLayout" name="horizontalLayout_3">
     <item>
      <widget class="QRadioButton" name="radHazard">
       <property name="toolTip">
        <string>A hazard is a situation that poses a level of threat to life, health, property, or environment. (Wikipedia)</string>
       </property>
       <property name="text">
        <string>Hazard</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QRadioButton" name="radExposure">
       <property name="toolTip">
        <string>Where people and property are situated.</string>
       </property>
       <property name="text">
        <string>Exposure</string>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Subcategory</string>
     </property>
     <property name="buddy">
      <cstring>cboSubcategory</cstring>
     </property>
    </widget>
   </item>
   <item row="2" column="1" colspan="2">
    <widget class="QComboBox" name="cboSubcategory">
     <property name="toolTip">
      <string>A subcategory represents the type of hazard.</string>
     </property>
    </widget>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QPushButton" name="pbnAdvanced">
     <property name="text">
      <string>Show advanced</string>
     </property>
     <property name="checkable">
      <bool>true</bool>
     </property>
    </widget>
   </item>
   <item row="4" column="0" colspan="3">
    <widget class="QGroupBox" name="groupBox">
     <property name="title">
      <string>Advanced editor</string>
     </property>
     <layout class="QGridLayout" name="gridLayout">
      <item row="1" column="0">
       <widget class="QFrame" name="frame">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label_4">
           <property name="text">
            <string>Keyword</string>
           </property>
           <property name="buddy">
            <cstring>cboKeyword</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cboKeyword">
           <item>
            <property name="text">
             <string>Subcategory</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Units</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Density</string>
            </property>
           </item>
           <item>
            <property name="text">
             <string>Datatype</string>
            </property>
           </item>
          </widget>
         </item>
         <item>
          <widget class="QLabel" name="label_5">
           <property name="text">
            <string>Value</string>
           </property>
           <property name="buddy">
            <cstring>cboValue</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="cboValue">
           <property name="editable">
            <bool>true</bool>
           </property>
           <property name="insertPolicy">
            <enum>QComboBox::InsertAtTop</enum>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QPushButton" name="pbnAddToList1">
           <property name="text">
            <string>Add to list</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="4" column="0">
       <widget class="QLabel" name="label_8">
        <property name="text">
         <string>Current keywords</string>
        </property>
        <property name="buddy">
         <cstring>lstKeywords</cstring>
        </property>
       </widget>
      </item>
      <item row="5" column="0">
       <widget class="QListWidget" name="lstKeywords">
        <property name="alternatingRowColors">
         <bool>true</bool>
        </property>
        <property name="selectionMode">
         <enum>QAbstractItemView::ExtendedSelection</enum>
        </property>
       </widget>
      </item>
      <item row="6" column="0">
       <widget class="QPushButton" name="pbnRemove">
        <property name="text">
         <string>Remove selected</string>
        </property>
       </widget>
      </item>
      <item row="0" column="0">
       <widget class="QRadioButton" name="radPredefined">
        <property name="text">
         <string>Predefined</string>
        </property>
       </widget>
      </item>
      <item row="3" column="0">
       <widget class="QFrame" name="frame_2">
        <property name="frameShape">
         <enum>QFrame::StyledPanel</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_2">
         <item>
          <widget class="QLabel" name="label_6">
           <property name="text">
            <string>Key</string>
           </property>
           <property name="buddy">
            <cstring>leKey</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leKey"/>
         </item>
         <item>
          <widget class="QLabel" name="label_7">
           <property name="text">
            <string>Value</string>
           </property>
           <property name="buddy">
            <cstring>leValue</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="leValue"/>
         </item>
         <item>
          <widget class="QPushButton" name="pbnAddToList2">
           <property name="text">
            <string>Add to list</string>
           </property>
          </widget>
         </item>
        </layout>
       </widget>
      </item>
      <item row="2" column="0">
       <widget class="QRadioButton" name="radUserDefined">
        <property name="text">
         <string>User Defined</string>
        </property>
       </widget>
      </item>
     </layout>
     <zorder>radPredefined</zorder>
     <zorder>frame</zorder>
     <zorder>radUserDefined</zorder>
     <zorder>frame_2</zorder>
     <zorder>label_8</zorder>
     <zorder>lstKeywords</zorder>
     <zorder>pbnRemove</zorder>
     <zorder>pbnAddToList1</zorder>
     <zorder>label_4</zorder>
     <zorder>cboKeyword</zorder>
     <zorder>cboValue</zorder>
     <zorder>label_5</zorder>
    </widget>
   </item>
   <item row="5" column="0" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
   <item row="0" column="0" colspan="3">
    <widget class="QLabel" name="lblLayerName">
     <property name="text">
      <string/>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <tabstops>
  <tabstop>radHazard</tabstop>
  <tabstop>radExposure</tabstop>
  <tabstop>cboSubcategory</tabstop>
  <tabstop>pbnAdvanced</tabstop>
  <tabstop>radPredefined</tabstop>
  <tabstop>cboKeyword</tabstop>
  <tabstop>cboValue</tabstop>
  <tabstop>pbnAddToList1</tabstop>
  <tabstop>radUserDefined</tabstop>
  <tabstop>leKey</tabstop>
  <tabstop>leValue</tabstop>
  <tabstop>pbnAddToList2</tabstop>
  <tabstop>lstKeywords</tabstop>
  <tabstop>pbnRemove</tabstop>
  <tabstop>buttonBox</tabstop>
 </tabstops>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>RiabKeywordsDialogBase</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>257</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>RiabKeywordsDialogBase</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>325</x>
     <y>501</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>pbnAdvanced</sender>
   <signal>toggled(bool)</signal>
   <receiver>groupBox</receiver>
   <slot>setVisible(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>151</x>
     <y>89</y>
    </hint>
    <hint type="destinationlabel">
     <x>120</x>
     <y>114</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radPredefined</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>42</x>
     <y>142</y>
    </hint>
    <hint type="destinationlabel">
     <x>50</x>
     <y>161</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radPredefined</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_2</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>73</x>
     <y>146</y>
    </hint>
    <hint type="destinationlabel">
     <x>75</x>
     <y>242</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radUserDefined</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame_2</receiver>
   <slot>setEnabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>69</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>114</x>
     <y>243</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>radUserDefined</sender>
   <signal>toggled(bool)</signal>
   <receiver>frame</receiver>
   <slot>setDisabled(bool)</slot>
   <hints>
    <hint type="sourcelabel">
     <x>38</x>
     <y>224</y>
    </hint>
    <hint type="destinationlabel">
     <x>30</x>
     <y>174</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
